insecure/corruptible/libp2p_node_factory.go

```diff
@@ -0,0 +1,34 @@
+package corruptible
```

Comment:
I feel that these corrupt node builder files should be in a separate package, not in `insecure/corruptible`. That package is for the corrupt network code (and that package should probably be renamed). I feel like all the corrupt node code should be in a new package like `insecure/node`